NotificationBanner/Classes/StatusBarNotificationBanner.swift

```diff
@@ -88,6 +74,9 @@ open class StatusBarNotificationBanner: BaseNotificationBanner {
         super.init(coder: aDecoder)
     }
 
+    override func basicHeight() -> CGFloat {
+        return 20.0 + basicYOffset
```

Comment:
Is this `20.0` the height of the status bar? If so, it might be better to use `UIApplication.shared.statusBarFrame.size.height`...

Reply:
It should be 20 in the end, because in some orientation the height of the status bar is 0, and in some it also adds up the safe area offset, which we already calculate in with `basicYOffset`. I have moved it to a variable 👍